src/uct/ib/base/ib_device.h

```diff
@@ -75,11 +80,13 @@ enum {
  */
 typedef struct uct_ib_address {
     uint8_t            flags;
-    /* Following fields appear in this order (if specified by flags) :
+    /* Following fields appear in this order (if specified by flags) unless RoCE is used:
```

Comment:
we can just say that the full gid always appears last, as specified by flags. no need to say it's only for RoCE / IB.